### PR TITLE
feat: new format call `Properties Files`

### PR DESCRIPTION
**Pull request by Pengap**
issues: #3

### CODE REVIEW

File: lib/transfomer.class.ts [hunk 1]
```diff
@@ -191,3 +152,94 @@ export class Transformer {
...
+        const rowPrimaryKey = rObj['id'];
+
+        valObj[rowPrimaryKey] = rObj[recordKey];
+      }
+    }
+    return retObj;
+  }
+}
+
+class FormatFactory {
```
Comment (mr-kelly) [inline lib/transfomer.class.ts:216]: separate multi class?

File: tests/mock_config.json [hunk 2]
```diff
@@ -54,5 +54,28 @@
         }
       }
     ]
+  },
+  {
+    "dirName": "../generated/",
+    "fileName": "i18n_{{ language }}.properties",
```
Comment (mr-kelly) [inline tests/mock_config.json]: Change to *?
Because, No other variabls here?

**Review by mr-kelly — CHANGES_REQUESTED** [review]

File: lib/transfomer.class.ts (hunk 1 above)
Comment (Pengap) [reply lib/transfomer.class.ts:216]: yes，Inner class

File: tests/mock_config.json (hunk 2 above)
Comment (mr-kelly) [reply tests/mock_config.json]: "i18n_*.properties" ？

Comment (Pengap) [reply tests/mock_config.json]:
```suggestion
    "fileName": "i18n_*.properties",
```

**Review by mr-kelly — APPROVED** [review]

**Pull request merged**